operator/controllers/lemming_controller.go

```diff
@@ -69,10 +84,83 @@ func (r *LemmingReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 		return ctrl.Result{}, err
 	}
 
+	switch pod.Status.Phase {
+	case corev1.PodRunning:
+		lemming.Status.Phase = lemmingv1alpha1.Running
+	case corev1.PodFailed:
+		lemming.Status.Phase = lemmingv1alpha1.Failed
+	default:
+		lemming.Status.Phase = lemmingv1alpha1.Unknown
+	}
+	lemming.Status.Message = fmt.Sprintf("Pod Details: %s", pod.Status.Message)
+	if err := r.Status().Update(ctx, lemming); err != nil {
+		log.Error(err, "unable to update lemming status")
+		return ctrl.Result{}, err
+	}
+
 	return ctrl.Result{}, nil
```

Comment:
Is the empty result intentional?

Reply:
yeah, non-empty allows for special behavior